### PR TITLE
AFHS-386 Added phone/email mismatch checks to pending screen.

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

File: src/web/routes/application/steps/decision/get-decision-status.test.js [hunk 1]
```diff
@@ -86,7 +85,7 @@ const DUPLICATE_RESULT = undefined // A DUPLICATE response from the claimant ser
 
 test('getDecisionStatus() should return undefined if verification result has no matching decision', (t) => {
   // TODO these non matching results will need to be updated as stories for epic 'receive my decision' are completed
-  const nonMatchingResults = [{}, SUCCESSFUL_RESULT, PENDING_RESULT_PHONE_OR_EMAIL_MISMATCH]
+  const nonMatchingResults = [SUCCESSFUL_RESULT]
   nonMatchingResults.forEach(verificationResult => {
```
Comment (mikeSmith08) [inline src/web/routes/application/steps/decision/get-decision-status.test.js]: Minor - we can remove the `forEach` now.

File: src/web/routes/application/steps/decision/get-decision-status.test.js [hunk 2]
```diff
@@ -127,3 +126,20 @@ test(`getDecisionStatus() should return ${PENDING} if address mismatches`, (t) =
...
+
+test(`getDecisionStatus() should return ${PENDING} if email and/or phone mismatches`, (t) => {
+  const mismatchingAddressResults = [
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'not_matched', emailAddressMatch: 'matched' },
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'not_matched', emailAddressMatch: 'not_matched' },
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'not_matched', emailAddressMatch: 'not_set' },
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'not_set', emailAddressMatch: 'matched' },
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'not_set', emailAddressMatch: 'not_matched' },
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'matched', emailAddressMatch: 'not_set' },
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'matched', emailAddressMatch: 'not_matched' }
```
Comment (YetAnotherMatt) [inline src/web/routes/application/steps/decision/get-decision-status.test.js]: Can we add a couple with 'not_held' please?

File: src/web/routes/application/steps/decision/get-decision-status.test.js (hunk 1 above)
Comment (dwybourn) [reply src/web/routes/application/steps/decision/get-decision-status.test.js]: Done.

File: src/web/routes/application/steps/decision/get-decision-status.test.js (hunk 2 above)
Comment (dwybourn) [reply src/web/routes/application/steps/decision/get-decision-status.test.js]: Done.

**Pull request merged**